### PR TITLE
Updated x64 default OS stack size to 512 KB for jdk17+

### PR DESCRIPTION
**Pull request by Sreekala-Gopakumar**
Closes https://github.com/eclipse-openj9/openj9-docs/issues/959

Updated the Default settings for the OpenJ9 VM, Migration from 11 to 17, What's new topics.

Signed-off-by: Sreekala Gopakumar sreekala.gopakumar@ibm.com

### CODE REVIEW

**Comment by Sreekala-Gopakumar** [comment]
Jenkins doc stage

File: docs/version0.33.md [hunk 1]
```diff
@@ -50,6 +51,10 @@ This was already required for OpenJDK 17 and is now also required from 11.0.16 t
 
 Linux builds for all platforms now use gcc 10.3 instead of gcc 7.5. See the list of [build environments](openj9_support.md#build-environments).
 
+### ![Start of content that applies to Java 17 plus](cr/java17plus.png) Increase in the default operating system stack size on x64 platforms
+
+The x64 default operating system stack size on xlinux, xmac, and Windows is increased from 256 KB to 512 KB to accommodate vector support. You can change the operating system stack size by using the [-Xmso](xmso.md) option.
```
Comment (pshipton) [inline docs/version0.33.md:56]: `xlinux`, `xmac` aren't how we usually refer to these platforms in the docs.

File: docs/openj9_defaults.md [hunk 2]
```diff
@@ -81,7 +81,7 @@ The last 2 columns show whether the default setting can be changed by a command-
...
 |Stack size for OS Threads **(32-bit)**. Use `-Xmso<size>`           |256 KB    |256 KB    |N/A              |32 KB            |256 KB    |:fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>| :fontawesome-solid-times:{: .no aria-hidden="true"}<span class="sr-only">no</span> |
-|Stack size for OS Threads **(64-bit)**. Use `-Xmso<size>`           |512 KB    |256 KB (512 KB on PPC)    |256 KB           |256 KB           |1 MB      |:fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>| :fontawesome-solid-times:{: .no aria-hidden="true"}<span class="sr-only">no</span> |
+|Stack size for OS Threads **(64-bit)**. Use `-Xmso<size>`           |512 KB    |256 KB (512 KB on PPC) <br>![Start of content that applies to Java 17 plus](cr/java17plus.png) 512 KB   |256 KB <br>![Start of content that applies to Java 17 plus](cr/java17plus.png) 512 KB          |256 KB <br>![Start of content that applies to Java 17 plus](cr/java17plus.png) 512 KB          |1 MB      |:fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>| :fontawesome-solid-times:{: .no aria-hidden="true"}<span class="sr-only">no</span> |
```
Comment (pshipton) [inline docs/openj9_defaults.md:84]: The default only changed to 512KB on xlinux for jdk17+, and was already 512KB on plinux, zlinux still uses 256K no matter the version.

**Comment by Sreekala-Gopakumar** [comment]
Jenkins doc stage

**Comment by Sreekala-Gopakumar** [comment]
Jenkins doc stage

**Review by pshipton — APPROVED** [review]

**Pull request merged**